.github/workflows/ci-regenerate-fixtures.yml

```diff
@@ -0,0 +1,39 @@
+name: Test With Regenerate Fixtures
+
+on:
+  schedule:
+    - cron: '0 15 * * 1' # Every Monday 7AM PST 
```

Comment:
Would we ever have to respond to the result of this job, kill it, or prevent it from running? 7AM PST might be earlier than we're online some weeks, especially if that Monday is a holiday!

Reply:
Yea I put 7AM PST because it takes about an hour to run so it should be complete by 8AM. Also since we have some people on east coast that's 11AM which seems pretty safe. Maybe we can discuss this amongst the team

Reply:
Decided to change to Tuesday morning in case of monday holiday but still before most releases. And adding a badge to the Github repo so that failures aren't missed